movie-mlops-project/preprocessing/main.py

```
@@ -0,0 +1,22 @@
+import pandas as pd
```

Ruff (0.12.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)